infrastructure/main.tf

```diff
@@ -21,8 +21,8 @@ module "db-rd-location-ref-api" {
   location            = var.location
   subscription        = var.subscription
   env                 = var.env
-  postgresql_user     = "dbrdlocationref"
-  database_name       = "dbrdlocationref"
+  postgresql_user     = "locrefdata"
+  database_name       = "locrefdata"
```

Comment:
agin it should both sync..just doe't change db else it fails load.

Reply:
ok. I will change